tests/components/recorder/db_schema_16.py

```diff
@@ -84,7 +84,7 @@ class Events(Base):  # type: ignore
     context_user_id = Column(String(MAX_LENGTH_EVENT_CONTEXT_ID), index=True)
     context_parent_id = Column(String(MAX_LENGTH_EVENT_CONTEXT_ID), index=True)
 
-    __table_args__ = (
+    __table_args__ = (  # noqa: PIE794
```

Comment:
The variable is already defined in line 72, but with other values. I added the `noqa` because I don't know the side-effects.

Line 159 and 240 are the same respectively

Reply:
That seems like the safest change for now